integration_test/testutil_test.go

```diff
@@ -161,12 +162,9 @@ func (it *ITest) AssertPluginFromIndex(plugin, indexName string) {
 	it.t.Helper()
 
 	receiptPath := environment.NewPaths(it.Root()).PluginInstallReceiptPath(plugin)
-	r, err := receipt.Load(receiptPath)
-	if err != nil {
-		it.t.Fatalf("error loading receipt: %v", err)
-	}
-	if r.Status.Source.Name != indexName {
-		it.t.Errorf("wanted index '%s', got: '%s'", indexName, r.Status.Source.Name)
+	pluginReceipt := it.loadReceipt(receiptPath)
```

Comment:
```suggestion
	r := it.loadReceipt(receiptPath)
```
I'd prefer to stick with `r` here. The scope where this variable is used is very small.